### DISCUSSION TITLE
License to use in our SAAS product

### DISCUSSION THREAD

**Discussion by xoraingroup** [add2cal/add-to-calendar-button#334] [Q&A] [answered]
Hi,

  We have a product that does alot of things related to encoding and streaming. We have a built-in calender option as well for the user to create schedules for creating encoding of the videos. 

My question is it allowed to use this library in our product so that we can offer our customers a way to store information in their own google and other calenders as well based on the license.

We will not be selling this feature, our core business is scheduling video encoding and streaming only.

Thank you

**Comment by jekuer** [comment, marked as answer]
Hi @xoraingroup ,
From what you write, it feels ok.
Can you share a link to the product?

**Reply by xoraingroup** [reply]
Hi,

  Its not launched yet, but its similar to [melonapp](https://melonapp.com/) and a bit of mux.com

Thank you

**Reply by jekuer** [reply]
Hi @xoraingroup ,

This would be perfectly fine then.
Rule of thumb: If your product is not about event management while the button only plays a supporting sub-role, it is completely safe. In other cases usually too, but worth the chat.

Would love to read about your product once it has been published!

All the best,
Jens

**Reply by xoraingroup** [reply]
Thank you Jens, 

 Yes its like a very tiny sub-role just to help users to store information in their respective account calenders for their encoding and streaming.

Thank you once again for the information :)

**Comment by vincenz2015** [comment]
I am looking at creating a shopify product which provides various product enhancements such as marquee scroller, product / cart notifications, WhatsApp chat and add to calendar (thats where your script can come in) and more.  So, the product will be sold as Shopify product enhancement widgets.  Can I use your code in this? Happy to do attribution or pay something for license, just getting started! Thanks

**Reply by jekuer** [reply]
will answer at #432 - since this is somehow a duplicate here.
